lib/checkin_handler.py

```diff
@@ -186,7 +186,7 @@ def _check_in_to_flight(self) -> JSON:
         site = CHECKIN_URL + self.flight.confirmation_number
 
         logger.debug("Making GET request to check in")
-        response = make_request("GET", site, headers, info)
+        response = make_request("GET", site, headers, info, random_sleep=True)
```

Comment:
I’d like someone to test with a check-in to see how much this will affect the time it takes (preferably on a device that almost never has 403s) to make sure this doesn’t make check-ins take too long to go through. Unfortunately I won’t have a flight to test for a while.

Reply:
Although the second check-in didn't pass, [this comment](https://github.com/jdholtz/auto-southwest-check-in/issues/230#issuecomment-2192313132) shows that it doesn't seemingly affect the check-in time. More samples should be observed, but this verifies that it logically shouldn't make much of a difference. The check-in will still go through in the first 10ish seconds no matter if the script sleeps for 0.5 seconds or 1-3 seconds. From check-ins in the past, the fastest I've seen it is 5 seconds after the minute.